### PR TITLE
[AccountDeleter]Adding stubbed definition for missing DI objects.

### PR DESCRIPTION
**Pull request by ryuyu**
Stubs out a minimal FeatureFlagService (used by PackageService) to allow DI to proceed successfully.

### CODE REVIEW

**Review by loic-sharma — APPROVED** [review]
Would it be possible to add a test that verifies the DI container doesn't have issues at runtime?

**Review by joelverhagen — APPROVED** [review]

File: src/AccountDeleter/EmptyFeatureFlagService.cs [hunk 1]
```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Services.Entities;
+using System;
+using System.Collections.Generic;
```
Comment (zhhyu) [inline src/AccountDeleter/EmptyFeatureFlagService.cs:6]: nit: The order of dependencies?

**Review by zhhyu — APPROVED** [review]

**Comment by ryuyu** [comment]
> 
> 
> Would it be possible to add a test that verifies the DI container doesn't have issues at runtime?

How would i go about doing this? Just have a test that attempts to hydrate the full real objects? That would require a full configuration for test, which I'm not against, but would be more work. Or if there is a simpler way, this sounds like a great idea.

**Review by dannyjdev — APPROVED** [review]

**Comment by ryuyu** [comment]
I filed https://github.com/NuGet/NuGetGallery/issues/8700 for adding this test. I had a bit of difficulty immediately. I'll look into pulling it in when possible.

**Pull request merged**